api/src/main/java/org/openmrs/api/db/hibernate/HibernateFormDAO.java

```diff
@@ -408,6 +408,7 @@ public Integer getFormCount(String partialName, Boolean published, Collection<En
 	 * @param containingAllFormFields
 	 * @param fields
 	 * @return
+	 * @should return criteria if containingAnyFormField is not empty.
```

Comment:
@hilz041 I dont think we add punctuations to these shoulds at least no point at the end
see https://wiki.openmrs.org/display/docs/Unit+Testing+With+at-should+Annotation
@dkayiwa please correct me if Im wrong :)